src/ShowPage/ShowPage.js

```diff
@@ -24,40 +21,48 @@ class ShowPage extends Component {
 
   determineSingleMovie = (id) => {
     getSingleMovie(id)
-    .then(singleMovie => this.setState({currentMovie: singleMovie}))
-    .catch(error => this.setState({ error: error }));
+      .then(singleMovie => this.setState({currentMovie: singleMovie.movie}))
+      .catch(error => this.setState({ error: error }));
   }
 
     getGenres = () => {
-      return this.props.genres.map(genre => {
-        return <p>genre.name</p>
+      return this.state.currentMovie.genres.map(genre => {
+        return <p>{genre}</p>
       })
     }
 
     render() {
-      console.log('showPage props', this.props);
-      console.log("ShowPage")
+      let userView;
+      if (this.state.currentMovie.overview) {
```

Comment:
Maybe we could switch these around so that the loading message exists as part of the `if` and the info from the fetch request exists as part of the `else`, might be easier to read.  For instance, we could do something like this:
```
  if (!this.state.currentMovie.overview) { 
     //loading message 
   } else {
      //content shows for movie when fetch request resolves
   }
```